results-processor/tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py3
+# We don't have or need setup.py for now.
+skipsdist=True
+
+[testenv]
+deps = -rrequirements.txt
```

Comment:
This look suspiciously like a typo?

Reply:
Actually, no, though I agree it's weird... It's in fact equivalent to `-r requirements.txt` in the command line but somehow `tox` doesn't like the space (the syntax is documented in one of its basic examples).